### PR TITLE
Fix flaky unit test for the CMReleasesContainer component

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

I tried to find a way to fix the flaky unit test in the content releases

### Why is it needed?

To pass the CI step

### CODE REVIEW

File: packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx [hunk 1]
```diff
@@ -59,8 +59,10 @@ describe('CMReleasesContainer', () => {
   it('should render the container', async () => {
     render();
 
-    const informationBox = await screen.findByRole('complementary', { name: 'Releases' });
-    expect(informationBox).toBeInTheDocument();
+    await waitFor(async () => {
```
Comment (markkaylor) [inline packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx:62]: I think findBy queries already use a waitFor https://testing-library.com/docs/dom-testing-library/api-async#findby-queries. I think what you end up doing here with an additional waitFor is increase the timeout / retries. But It looks like you can pass waitFor options to the findBy query to do that as well.

Comment (joshuaellis) [reply packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx:62]: Mark is right, it'll also fail if it's not in the document. There's probably a different `async` fetch happening elsewhere that updates the component, you might needed to await something else.

**Comment by joshuaellis** [comment]
since @remidej merged https://github.com/strapi/strapi/pull/20250, can we close this?

**Comment by simotae14** [comment]
Already fixed in https://github.com/strapi/strapi/pull/20250

**Pull request closed without merging**